misk-hibernate-testing/src/main/kotlin/misk/jdbc/TestDatabasePool.kt

```diff
@@ -140,7 +146,11 @@ class TestDatabasePool(
       val evictBefore = LocalDate.now(clock).minus(Period.ofDays(retention.toDays().toInt()))
```

Comment:
This will use a Fake lock in tests I think. Is the behavior still intentional?

Reply:
Actually it uses `Clock.systemUTC()`!